src/Sulu/Bundle/SnippetBundle/Domain/Event/DefaultSnippetModifiedEvent.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SnippetBundle\Domain\Event;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+use Sulu\Bundle\SnippetBundle\Admin\SnippetAdmin;
+use Sulu\Bundle\SnippetBundle\Document\SnippetDocument;
+
+class DefaultSnippetModifiedEvent extends DomainEvent
```

Comment:
think i would call this `WebspaceDefaultSnippetModifiedEvent` because all other events are prefixed with the resource too. or maybe we should use another resource-key? 🙈

Reply:
I thought we agreed on using `webspaces` as resource, so I renamed it 🙈

Reply:
sounds good to me 🙂